server/api/api/endpoints/alerts.py

```diff
@@ -71,6 +72,7 @@ async def store_alert(
         project,
         name,
         alert_data,
+        force_reset,
```

Comment:
I don't understand something. In httpdb you used `bool2str` which converts this to "yes" or "no" so how is this a bool here? Do you need to use `strtobool` like `with_mlrun = strtobool(data.get("with_mlrun", "on"))`
I see the same method is used in other places like `list_runs` I wonder if this is a bug

Reply:
I checked that out and seems like the FastApi converts the "yes" or "no", to True or "False"